components/buttonCustom/ButtonCustom.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from "react";
+import { styles } from "./styles";
+import { Colors } from "../../colors/Colors";
+import { TouchableHighlight, Text, View } from "react-native";
+
+export class ButtonCustom extends Component {
```

Comment:
Please name the component just `Button`

Reply:
For cases where we don't have any logic in the component, it's better to use a presentational component:
```js
export const Button = ({ }) => (
<JSX />
)
```

Reply:
Also, since it's inside a directory, you could name the file `index.js` so it's easier to import when you want to use it later on